src/RawModelExportItem.tsx

```diff
@@ -0,0 +1,68 @@
+import Divider from '@mui/material/Divider'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/system/Box'
+import Button from '@mui/material/Button'
+
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import DirectoryTreeView from './common/DirectoryTreeView'
+import SplitButton from './common/SplitButton'
+
+function RawModelExportItem({ deploymentUuid, version }: { deploymentUuid: string; version: any }) {
+  const router = useRouter()
+  const [displayTree, setDisplayTree] = useState<boolean>(false)
```

Comment:
Typing isn't really necessary for strings and booleans (applies to line below too!)